### PR TITLE
Add section about using R-Ladies in grant proposals

### LINKED ISSUE #73: Add section about using R-Ladies in grant proposals
There was a case where the R-Ladies name was used in a grant proposal to potentially help secure funding (promising benefits to R-Ladies).  However, this was done without any coordination or communication with the R-Ladies Global team.

Let's add a section in [Partnerships](https://guide.rladies.org/organization/intro/partnerships/) (or maybe [Allies](https://guide.rladies.org/ally/)?) to let people know they should contact us at info _at_ rladies.org when doing this so we know about it and can provide input/approval.  We want to make sure that it's an organization that is in alignment with our mission.

### PR DESCRIPTION
**Pull request by maelle**
Fix #73

### CODE REVIEW

**Comment by maelle** [comment]
@shannonpileggi this would be merged after your conference participation PR as it adds a link to the page you created.

File: content/organization/intro/partnerships/index.en.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ You can also look or ask whether the organization includes and has been supporti
 
 If you are invited to give a talk,
 * If you are invited to talk about a topic other than R-Ladies, make sure that you are invited because of your expertise and not just your relationship to R-Ladies.
-* If you are invited to talk about R-Ladies, it can be a great opportunity - feel free to take advantage! 
+* If you are invited to talk about R-Ladies, refer to our [specific guidance](/comm/conferences/), in particular get in touch with the global team.
```
Comment (yabellini) [inline content/organization/intro/partnerships/index.en.md:20]:
```suggestion
* If you are invited to talk about R-Ladies, refer to our [specific guidance](/comm/conferences/), in particular get in touch with the global team (`info@rladies.org`).
```

**Review by yabellini — APPROVED** [review]
I think this is a good addition. Thanks!

**Review by ledell — APPROVED** [review]
@maelle This looks good, should it be merged? (sorry for late reply)

**Pull request merged**